t_frang/test_request_rate_burst.py

```diff
@@ -0,0 +1,267 @@
+"""Tests for Frang directive `request_rate` and 'request_burst'."""
+import time
+
+from t_frang.frang_test_case import ONE, ZERO, FrangTestCase
+
+DELAY = 0.125
+ERROR_MSG = 'Warning: frang: request {0} exceeded for'
+ERROR_MSG_BURST = 'Warning: frang: requests burst exceeded'
+
+
+class FrangRequestRateTestCase(FrangTestCase):
+    """Tests for 'request_rate' and 'request_burst' directive."""
+
+    clients = [
+        {
+            'id': 'curl-1',
+            'type': 'external',
+            'binary': 'curl',
+            'cmd_args': '-Ikf -v http://127.0.0.4:8765/ -H "Host: tempesta-tech.com:8765"',
+        },
+    ]
+
+    tempesta = {
+        'config': """
+            frang_limits {
+                request_rate 4;
```

Comment:
It makes sense to test `request_rate` and `request_burst` in single configuration and trigger each of the limits in the test cases.

Reply:
ok, added